Replace "instructor-of-record" with better understandable term (#1367)
carpentries/instructor-training: _episodes/01-welcome.md

> 1.  I have been a graduate or undergraduate teaching assistant for a university/college course.
> 2.  I have not had any teaching experience in the past.
> 3.  I have taught a seminar, workshop, or other short or informal course.

> > 4.  I have been the main/responsible teacher for a course at my own university/college course.

Reviewer: ```suggestion
> 4.  I have been the primary or responsible teacher for a university/college course.
```
